tidalapi/page.py

```diff
@@ -179,12 +230,12 @@ def show_more(self):
 class FeaturedItems(PageCategory):
     """Items that have been featured by TIDAL."""
 
-    items: Optional[list["PageItem"]] = None
+    items: Optional[List["PageItem"]] = None
 
-    def __init__(self, session):
+    def __init__(self, session: "Session"):
         super(FeaturedItems, self).__init__(session)
```

Comment:
not in your pr but should be `super().__init__(session)`. (this is just old code)

Reply:
Yeah, will make the change since I've been stripping out new-style class declarations anyway.